### PR TITLE
chore: Extract UIButton and UILabel style application

### PR DESCRIPTION
**Pull request by kaphacius**
This PR addes helper method and initializers to `UILabel` and `UIButton` that work with our defined `TextStyle` and `ButtonStyle`

### CODE REVIEW

File: Adyen/Helpers/UILabelHelpers.swift [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    
+    /// Applies given `TextStyle` to the UILabel
+    /// Sets `adjustsFontForContentSizeCategory` to `true`
+    /// - Parameter style: `TextStyle` to be applied
+    internal func apply(_ style: TextStyle) {
+        base.font = style.font
+        base.textColor = style.color
+        base.textAlignment = style.textAlignment
+        base.backgroundColor = style.backgroundColor
+        round(using: style.cornerRounding)
```
Comment (descorp) [inline Adyen/Helpers/UILabelHelpers.swift:33]: better remove this 🤔

**Review by mohammedDehairy — APPROVED** [review]

File: Adyen/UI/Form/Items/Error/FormErrorItemView.swift [hunk 2]
```diff
@@ -47,18 +47,12 @@ internal final class FormErrorItemView: FormItemView<FormErrorItem>, Observer {
     // MARK: - Message
 
     private lazy var messageLabel: UILabel = {
-        let messageLabel = UILabel()
-        messageLabel.font = item.style.message.font
-        messageLabel.adjustsFontForContentSizeCategory = true
-        messageLabel.textColor = item.style.message.color
-        messageLabel.textAlignment = item.style.message.textAlignment
-        messageLabel.backgroundColor = item.style.message.backgroundColor
+        let messageLabel = UILabel(style: item.style.message)
```
Comment (descorp) [inline Adyen/UI/Form/Items/Error/FormErrorItemView.swift:50]: nice :D

File: Adyen/Helpers/UILabelHelpers.swift (hunk 1 above)
Comment (kaphacius) [reply Adyen/Helpers/UILabelHelpers.swift:33]: why? 🤔

File: Adyen/UI/Views/SubmitButton.swift [hunk 3]
```diff
@@ -61,14 +61,8 @@ public final class SubmitButton: UIControl {
...
     private lazy var titleLabel: UILabel = {
-        let titleLabel = UILabel()
-        titleLabel.font = style.title.font
-        titleLabel.adjustsFontForContentSizeCategory = true
-        titleLabel.textColor = style.title.color
-        titleLabel.backgroundColor = style.title.backgroundColor
-        titleLabel.textAlignment = style.title.textAlignment
+        let titleLabel = UILabel(style: style.title)
         titleLabel.isAccessibilityElement = false
-        titleLabel.translatesAutoresizingMaskIntoConstraints = false
```
Comment (descorp) [inline Adyen/UI/Views/SubmitButton.swift:71]: we probably still need this line 😅

File: Adyen/Helpers/UILabelHelpers.swift (hunk 1 above)
Comment (descorp) [reply Adyen/Helpers/UILabelHelpers.swift:33]: as I mentioned in mattermost - `round` can be percent-based - it is calculated based on real sizes of the element. If element is not yet set (layoutSubviews never called) the size is `.zero` so will be the rounding..

**Pull request merged**